blocks/donation-form/class-give-donation-form-block.php

```diff
@@ -172,6 +178,14 @@ public function render_donation_form( $attributes ) {
 		$parameters['display_style']         = $attributes['displayStyle'];
 		$parameters['continue_button_title'] = trim( $attributes['continueButtonTitle'] );
 
+		$class_names = Html::classNames(
+			isset( $attributes['className'] ) ? $attributes['className'] : null
+		);
+
+		add_filter( 'give_form_wrap_classes', function() use ( $class_names ) {
```

Comment:
`give_form_wrap_classes ` filter hook returns list of classes in `array` format. 

The current implementation will override other classes and only return a list of custom classes which can cause of unexpected output on the frontend. Can you update your logic to merge custom classes with the list of classes.

Reply:
Thanks! I forgot to merger.